frame/transaction-storage/README.md

```diff
@@ -1,8 +1,63 @@
 # Transaction Storage Pallet
 
 Indexes transactions and manages storage proofs.
-# Transaction Storage Pallet
 
-Indexes transactions and manages storage proofs.
+Allows storing arbitrary data on the shain. Data is automaticcaly removed after `StoragePeriod` blocks, unless the storage is renewed.
+Validators must submit proof of storing a random chunk of data for block `N - StoragePeriod` when producing block `N`.
+
+# Runing a chain
+
+The following describes how to setup a new storage chain.
+
+Start with generating a chain spec.
+```
+cargo run --release -- build-spec --chain=local > sc_init.json
+```
+Edit the json chain spec file to customise the chain. The storage chain genesis params are configured in the `transactionStorage` section.
+Note that `storagePeriod` is specified in blocks and changing it also requires code changes at the moment.
```

Comment:
Could be good to have a sample of chain spec with the specified sections.

Reply:
Generated default is the sample. All the values are there.